src/cljs/snapshot/lumo/js_deps.cljs

```diff
@@ -35,13 +35,27 @@
                              conj munged-ns)))
                {:requires [] :provides []})))
 
-(defn parse-lib
-  "Converts a closure lib path into a structure which describes the module."
+(defn all-files
+  "If the given file is not a directory, returns a list containing the file,
```

Comment:
Let's use `file-seq` from `src/bundled/util` here. (it's fine to duplicate it in `src/snapshot/common`)